### ISSUE TITLE
confuse about the txt file

### ISSUE THREAD

**Issue by cocovoc** [VisionLearningGroup/DANCE#7]
In txt file, I find paths of some images from /research/masaito/, what the difference between images in /research/masaito/ and data/

**Comment by take2rohit** [comment]
Yeah facing a similar issue. There is a lot of problem with paths.

**Comment by AeRabelais** [comment]
For anyone who may be using the code going forward and runs into this problem -

The text files that have directories that begin with **research/masaito** should begin with *data*. It's not expressly noted, but in the _utils_ folder, there's a file called _txt_edit.py_ that makes this change for you.
